Framework/Algorithms/src/CompareWorkspaces.cpp

```diff
@@ -1269,25 +1274,19 @@ void CompareWorkspaces::doTableComparison(const API::ITableWorkspace_const_sptr
   const bool checkAllData = getProperty("CheckAllData");
   const bool isRelErr = getProperty("ToleranceRelErr");
   const double tolerance = getProperty("Tolerance");
-  bool mismatch = false;
+  bool mismatch;
```

Comment:
Either CppCheck or clang-format insisted I do this.  But I agree with you, it's safer to set it to something.

Reply:
clang-format only messes with whitespace. I'd suppress cppcheck as a false positive.

Reply:
I don't think there's any real risk in the case of an empty table.  If the table is empty, it will just skip over the for loop and do nothing.  This value isn't used for anything after for the for loop/